modules/test/playwright/pages/portal-search-web/SearchPage.ts

```diff
@@ -151,6 +151,12 @@ export class SearchPage {
 		await expect(this.page.locator('#modalIframe')).toBeVisible();
 	}
 
+	async savePortletConfiguration() {
+		await this.modalIFrame.getByRole('button', {name: 'Save'}).click();
+
```

Comment:
It might be a good idea to add a `await waitForSuccessAlert(this.page);` after this. 

You can look for usages of this util method. https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/utils/waitForSuccessAlert.ts